test/images/scripts/lib/debian.install

```diff
@@ -46,6 +46,9 @@ if [ -n "$do_build" ]; then
     tar -xzf cockpit-*.tar.gz
     ( cd cockpit-*/
       cp -rp tools/debian debian
+      # Hack: Remove PCP build dependencies while pcp is not in testing
+      # (https://tracker.debian.org/pcp)
+      sed -i '/libpcp.*-dev/d' debian/control
```

Comment:
The continuous delivery scripts are here:

https://github.com/cockpit-project/cockpituous/blob/master/release/release-dsc

https://github.com/cockpit-project/cockpituous/blob/master/release/release-debian

https://github.com/cockpit-project/cockpituous/blob/master/release/release-ubuntu-ppa

There's very little Cockpit specific in the Debian continuous delivery scripts, and we should be working towards removing the last remaining Cockpit specific bits.